### DISCUSSION TITLE
Best satellite imagery?

### DISCUSSION THREAD

**Discussion by zakmar06** [kubilus1/autoortho#166] [Q&A] [answered]
I'm currently getting a lot of CTDs (VK_Out_of_memory) with Bing Maps imagery 

What do other people use?

What is the best for quality and performance?

**Comment by jonaseberle** [comment]
If you go out of memory, switching imagery providers won't help.

**Comment by BinaryOutlook** [comment, marked as answer]
VK_Out_of_memory means that you should either optimize your system during Xplane or get more memory sticks with the current low price. Is cheaper to get 32GB ddr4 than a Toliss jet today.

**Reply by zakmar06** [reply]
Totally aware of that.
I have 32GB of RAM and 16 of VRAM
Log text:
:06:14.602 E/GFX/VK: Failed to allocate from memory index. Requirements: { 79066760, 4, f }
0:06:14.602 E/GFX/VK: Used memory type index: 0 (VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT)
0:06:14.602 E/GFX/VK: Memory heap Usage: 5647686144. Budget: 16071197696
0:06:14.602 E/GFX/VK: Encountered error: VK_ERROR_OUT_OF_DEVICE_MEMORY

I guess there is an unoptimized add-on in my sim rather than my specs.
